WordPress-iOS-Shared/Core/WPStyleGuide.h

```diff
@@ -27,48 +27,65 @@
 + (UIFont *)tableviewSubtitleFont;
 + (UIFont *)tableviewSectionHeaderFont;
 
-// Colors
+// Color
 + (UIColor *)wordPressBlue;
-+ (UIColor *)baseLighterBlue;
-+ (UIColor *)baseDarkerBlue;
 + (UIColor *)lightBlue;
-+ (UIColor *)newKidOnTheBlockBlue;
-+ (UIColor *)midnightBlue;
++ (UIColor *)mediumBlue;
++ (UIColor *)darkBlue;
++ (UIColor *)gray;
++ (UIColor *)lightGrey;
```

Comment:
Just a note: I don't have a preference between `gray` vs `grey` but let's change them all to the same thing. In the handbook and on the web, we use `gray`, but on Android, everything is grey.